.github/workflows/helm-publish.yml

```diff
@@ -9,10 +9,18 @@ jobs:
     if: ${{ github.event.label.name == 'helm_release' }}
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - name: Release helm chart
-        uses: J12934/helm-gh-pages-action@master
+      - name: Checkout repo
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
+
+      - name: Install Helm
+        uses: azure/setup-helm@5119fcb9089d432beecbf79bb2c7915207344b78 # v3.5
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@be16258da8010256c6e82849661221415f031968 # v1.5.0
         with:
-          charts-folder: charts
-          deploy-branch: helm-release
-          access-token: ${{ secrets.HELM_PUSH_TOKEN }}
\ No newline at end of file
+          charts_dir: charts
+          pages_branch: helm-release
+          mark_as_latest: false
```

Comment:
Do we need this?

Reply:
I think we do or else it will try to publish a tag and fail here:
https://github.com/smartcontractkit/chainlink/actions/runs/6628870377/job/18006919805?pr=11062

```
Error: error creating GitHub release chainlink-cluster-0.1.3: POST https://api.github.com/repos/smartcontractkit/chainlink/releases: 422 Validation Failed [{Resource:Release Field:pre_receive Code:custom Message:pre_receive protected tag 'chainlink-cluster-0.1.3' check failed:
  You're not authorized to create a tag} {Resource:Release Field: Code:custom Message:Published releases must have a valid tag}]
```